demo/src/Components/Pills/Pills.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import T from 'prop-types'
+import classNames from 'classnames'
+
+import List from './List'
+import Panels from './Panels'
+
+class Pills extends Component {
+  constructor() {
+    super()
+
+    this.state = {
+      activeIndex: 0,
```

Comment:
Yes thanks - I tried this reverting to false and it was breaking so left as is - most of these are fixed now - have left the defaultProp one as have to declare values for each - the only rule I have changed is the one re multiline props so once this gets merged will fixed conflicts on other branch I'm working on.